tests/cli/test_cmds_logs.py

```diff
@@ -755,3 +772,135 @@ def test_get_log_reader():
 
         actual = logs.get_log_reader()
         assert isinstance(actual, logs.ScribeLogReader)
+
+
+def test_generate_start_end_time():
+    start_time, end_time = logs.generate_start_end_time()
+
+    # Default for no args, test that there's a 30 minute difference
+    time_delta = end_time - start_time
+
+    # Do a loose comparison to make this test less time sensitive
+    actual_time = time_delta.total_seconds()
+    ideal_time = 30 * 60
+
+    assert abs(actual_time - ideal_time) < 0.1
```

Comment:
There's a comment right above it:

```python
# Do a loose comparison to make this test less time sensitive
```

If it's == 0 it will fail on slower build platforms like travis.ci where lines of code aren't executed as instantly as you'd expect